### PR TITLE
Update and rename bug_report.md to bug_report.yml

### PR DESCRIPTION
**Pull request by bud3699**
Just switches your issue template to be more interactive
This aint a contribution, just the issue template was scuffed

image
image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: Which platform are you playing the game on?
+      options:
+        - Steam
+        - Epic Games
+        - Xbox Game Pass
```
Comment (ismail0234) [inline .github/ISSUE_TEMPLATE/bug_report.yml:24]: XBOX Version is not supported. You can remove this line.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to Reproduce
+      description: List the steps to reproduce the bug.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '...'
+        3. See error
```
Comment (ismail0234) [inline .github/ISSUE_TEMPLATE/bug_report.yml:44]: This step should be mandatory.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    validations:
+      required: false
+  - type: textarea
+    id: log-files
+    attributes:
+      label: Log Files
+      description: Attach the log files or explain how to retrieve them.
+      placeholder: Drag and drop log files here or provide the details.
+    validations:
+      required: false
```
Comment (ismail0234) [inline .github/ISSUE_TEMPLATE/bug_report.yml:54]: This step should be mandatory.

**Comment by bud3699** [comment]
Changed it, I didnt think gamepass worked due to not being able to modify xbox games if owned on gamepass, but it was there before so I thought i'd add it

**Pull request merged**